### PR TITLE
added tintColor prop for iOS

### PR DESCRIPTION
**Pull request by szalaybalazs**
### Does any other open PR do the same thing?

No

(please answer here)

### What issue is this PR fixing?

It does not fix any issue, but adds a new feature, the iOS tintColor prop.
Propname: `tintColor`
Proptype: `UIColor`
optional: true

iOS only, for the moment

### How did you test this PR?

Tested on simulator, and on real device as well.

### CODE REVIEW

**Comment by rborn** [comment]
@szalaybalazs thanks a lot for the PR. 
Could you please explain more what's this doing and also amend the docs so users know how to use it and what are the limitations ?  (ios/applemap only if is the case )

😻

**Comment by szalaybalazs** [comment]
@rborn I have updated the readme, including all the limitations (iOS/applemaps)

Thanks in advance

**Comment by rborn** [comment]
Ok great, one comment only and we're good 😻

**Comment by szalaybalazs** [comment]
Sorry, I got a bit lost. What more comment do you need?

The tintColor on iOS changes the the color of the UI elements of the mapView. (https://developer.apple.com/documentation/mapkitjs/mapkit/map/2973946-tintcolor)

**Comment by rborn** [comment]
Sorry,  I forgot to hit "start review" 😅 , is the typo above

File: docs/mapview.md [hunk 1]
```diff
@@ -41,6 +41,7 @@
 | `loadingEnabled` | `Boolean` | `false` | If `true` a loading indicator will show while the map is loading.
 | `loadingIndicatorColor` | `Color` | `#606060` | Sets loading indicator color, default to `#606060`.
 | `loadingBackgroundColor` | `Color` | `#FFFFFF` | Sets loading background color, default to `#FFFFFF`.
+| `tintColor` | `color` | `null` | Sets the tint color of the map. (Changes the color of the postiion indicator) Defaults to system blue. **Note:** iOS (Apple maps) only.
```
[the remark this answers is not in the document]
Comment (szalaybalazs) [reply docs/mapview.md]: Fixed it, sorry!

**Comment by rborn** [comment]
@szalaybalazs thanks 😻

**Comment by AndrewSB** [comment]
Is this usable from the javascript props? I don't see an edit to `components/MapView.js` to include into `MapViewProps`

**Pull request merged**